pkg/serviceimport/map.go

```diff
@@ -108,9 +109,7 @@ func (m *Map) Put(serviceImport *lighthousev2a1.ServiceImport) {
 			}
 		}
 
-		for _, info := range serviceImport.Status.Clusters {
-			remoteService.clusterIPs[info.Cluster] = info.IPs
-		}
+		remoteService.clusterIPs[serviceImport.GetLabels()[lhconstants.LabelSourceCluster]] = []string{serviceImport.Spec.IP}
```

Comment:
So it appears `clusterIPs` will only ever have 1 IP so do we need to store as an array? Also change `GetIPs` to return single IP?

Reply:
> So it appears `clusterIPs` will only ever have 1 IP so do we need to store as an array? Also change `GetIPs` to return single IP?

This was needed when we were using ServiceImport for headless. Now this can be changed to just a string.